openmetadata-ui/src/main/resources/ui/src/pages/AddCustomMetricPage/AddCustomMetricPage.tsx

```diff
@@ -268,4 +269,4 @@ const AddCustomMetricPage = () => {
   );
 };
 
-export default AddCustomMetricPage;
+export default withPageLayout('')(AddCustomMetricPage);
```

Comment:
Can we set title? it's responsible for meta info, also make sure to use localised one